books/dev-books/Vue/vue.md

```diff
@@ -8,3 +8,9 @@ Please kindly follow this structure when you are contributing
 // List them as a direct link to the resource (No redirects)
   
 - [30 Days of Vue](https://www.newline.co/30-days-of-vue)
+
+### G
+-[Getting to Know Vue js Learn to Build Single Page Applications in Vue from Scratch by Brett Nelson](https://drive.google.com/file/d/1GFJfSjTrJCmc6m5Xd9NpYwq65cbDu1K1/view?usp=share_link)
+
+
+### L
```

Comment:
Looks great. we would need to move the numeric titles to the bottom of the list.